src/behavior-considered-undefined.md

```diff
@@ -49,6 +49,8 @@ code.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
 * Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI.
+* Calling a foreign (e.g. C++) function that unwinds (`throw`s) via a function
+  declaration or pointer declared with a non-unwinding ABI such as `"C"`
```

Comment:
```suggestion
* Calling a foreign (e.g. C++) function that unwinds (`throw`s) via a function
  declaration or pointer declared with a non-unwinding ABI such as `"C"`
* Deallocating a Rust stack frame without executing destructors for local variables owned by that stack frame. This can occur with C functions like `longjmp`, with unwinding via C++ exceptions, or in other ways.
```

I'm not sure how clear this is, but it seems worth calling out -- unless it's already on the list somewhere.

Reply:
Unwinding with exceptions shouldn't be listed here. If it unwinds past `C-unwind` then it's fine, and if it unwinds past `extern "C"` then it's an UB per the previous clause.

Reply:
I don't see it on the list, so I'll add it. I do think it's a bit tangential to `"C-unwind"` being stabilized, especially with Gary's caveat, but it seems like it should be included in this list anyway.